app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Please make sure to stick to the given wireframe, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Modify%20the%20views%20in%20devise%20for%20registration%20and%20login%20to%20match%20the%20wireframes%20from%20the%20Sneak%20Peek%20and%20your%20styling.) requirement.

![image](https://user-images.githubusercontent.com/62290545/231511302-06ce1edd-0190-4d00-9f18-be90152b9e90.png)

Reply:
@ShadyShawkat Can you please specify what wire frame you are talking about, because as per the requirements, we are not supposed to add any design , but focus more on functionality, so please if i can see an image of the wire frame for a login page i would appreciate that

Reply:
Hi @shegy28, the requirement says the following: "Modify the views in `devise` for registration and login to match the wireframes". You're to stick to the wireframe of the login and registration pages to match [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_login.png) wireframe design.

Reply:
Hi @shegy28

Please note that the wireframes are in the [Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A).  Although styling is not required, in order to layout your elements in the page to match the wireframe you still need to make use of CSS attributes like text-align, margins, padding, or Flexbox and Grid.

Reply:
@KDlamini i have made the changes to match the wire frame